king_phisher/client/__init__.py

```diff
@@ -37,13 +37,18 @@
 repo = gi.Repository.get_default()
 repo.get_loaded_namespaces()
 
+if 'WebKit2' in repo.get_loaded_namespaces():
+        WebKitX = 'WebKit2'
+else:
+        WebKitX = 'WebKit'
+
 _gi_versions = [
 	('Gdk', '3.0'),
 	('Gtk', '3.0'),
 	('GtkSource', '3.0'),
 	('JavaScriptCore', ('3.0', '4.0')),
 	('Pango', '1.0'),
-	('WebKit2', ('3.0', '4.0'))
+        (WebKitX, ('3.0', '4.0'))
 ]
```

Comment:
Change this to an append on `_gi_versions`. Also if neither are available append WebKit2 so the error message implies the preference towards the newer version.